site/community.markdown

```diff
@@ -68,37 +68,35 @@ There are a number of conferences and events featuring Haskell, some focusing on
 
 *   [The Haskell Symposium](https://www.haskell.org/haskell-symposium/)
 *   [Haskell Implementors' Workshop](https://wiki.haskell.org/HaskellImplementorsWorkshop)
-*   [The International Conference on Functional Programming](http://www.icfpconference.org/)
+*   [The International Conference on Functional Programming](https://www.icfpconference.org/)
 *   [Symposium on Principles of Programming Languages](https://www.sigplan.org/Conferences/POPL/)
-*   [International Symposia on Implementation and Application of Functional Languages](http://www.ifl-symposia.org/)
-*   [Symposium on Trends in Functional Programming](http://www.tifp.org/)
+*   [International Symposia on Implementation and Application of Functional Languages](https://ifl-symposia.org/)
+*   [Symposium on Trends in Functional Programming](https://trendsfp.github.io/)
 
 ### Non-Academic Conferences
 
 *   [Commercial Users of Functional Programming (Roving)](http://cufp.org/)
-*   [Functional Conf (Bangalore, IN)](http://functionalconf.com/)
-*   [BOB (Berlin, DE)](http://bobkonf.de/)
-*   [Skills Matter's Haskell eXchange (London, GB)](https://skillsmatter.com/conferences/11741-haskell-exchange-2019)
-*   [LambdaConf (Boulder, CO, USA)](http://lambdaconf.us/)
-*   [Lambda World (Cádiz, Spain and Seattle, USA)](http://www.lambda.world/)
-*   [Compose :: Conference (NY, NY, USA)](http://composeconference.org)
-*   [LambdAle (London, GB)](https://lambdale.org/)
-*   [Haskell Love Conference (online)](https://haskell.love)
+*   [Functional Conf (Bangalore, IN)](https://functionalconf.com/)
+*   [BOB (Berlin, DE)](https://bobkonf.de/)
+*   [LambdaConf (Boulder, CO, USA)](https://www.lambdaconf.us/)
+*   [Lambda Days (Kraków, Poland)](https://www.lambdadays.org/)
+*   [Lambda World (Cádiz, Spain and Seattle, USA)](https://www.lambda.world/)
+*   [Compose :: Conference (NY, NY, USA)](https://www.composeconference.org/)
 
 ### Hackathons
 
 Haskell Hackathons are a long tradition, with lots of learning and social exchange. In many ways they function as semi-structured conferences. Here are some of the most notable:
 
+*   [ZuriHac (Zürich, CH)](https://zurihac.com)
 *   [BayHac (San Francisco Bay Area, CA, USA)](https://wiki.haskell.org/BayHac)
 *   [Hac Phi (Philadelphia, PA, USA)](https://wiki.haskell.org/Hac_Phi)
-*   [ZuriHac (Zurich, CH)](https://wiki.haskell.org/ZuriHac)
 *   [HacFreiburg (Freiburg, De)](https://wiki.haskell.org/HacFreiburg2017)
```

Comment:
Thanks! Would you mind adding `MuniHac` as well? Its website is https://munihac.de

Reply:
No worries - it's done.